frontend/app/services/wsaddress-service.server.ts

```diff
@@ -36,6 +36,15 @@ const validateAddressResponseSchema = z.object({
   }),
 });
 
+export interface CorrectAddressResponse {
```

Comment:
It's not needed since `correctAddress` is implicitly returning the type.

Reply:
Removed `CorrectAddressResponse` uses [here](https://github.com/DTS-STN/canadian-dental-care-plan/compare/3b8fa830bade1a4323acf74e1c574a7942d03d2c..0fb03103edb53651317d5b7096d9360c3bfb87d7#diff-a330c9ede4049228d6825394285f16e4e4737fb401eb33bdfaa0021111a6bb85) and added partial mocks [here](https://github.com/DTS-STN/canadian-dental-care-plan/compare/3b8fa830bade1a4323acf74e1c574a7942d03d2c..0fb03103edb53651317d5b7096d9360c3bfb87d7#diff-a72bc829ebe90af4d2b71eaa29c4482e8a870cfa4fbd9ffbb1760d2a4154f881).